gramps_webapi/api/resources/event.py

```diff
@@ -24,9 +24,9 @@ class EventResourceHelper(GrampsObjectResourceHelper):
     def object_extend(self, obj: Event, args: Dict) -> Event:
         """Extend event attributes as needed."""
         db_handle = self.db_handle
-        if args["profile"]:
+        if "profile" in args:
```

Comment:
Why is this no longer a boolean? Will `profile=0` be interpreted as `True`?

Reply:
If profile is not present in the parms then it is not generated, if ?profile is in the response it is generated. No need for =0 or =1. I think it is cleaner. Same for the others that were booleans.